doc/src/Howto_chunk.rst

```diff
@@ -58,28 +58,30 @@ chunk ID for an individual atom can also be static (e.g. a molecule
 ID), or dynamic (e.g. what spatial bin an atom is in as it moves).
 
 Note that this compute allows the per-atom output of other
-:doc:`computes <compute>`, :doc:`fixes <fix>`, and
-:doc:`variables <variable>` to be used to define chunk IDs for each
-atom.  This means you can write your own compute or fix to output a
-per-atom quantity to use as chunk ID.  See the :doc:`Modify <Modify>`
-doc pages for info on how to do this.  You can also define a :doc:`per-atom variable <variable>` in the input script that uses a formula to
-generate a chunk ID for each atom.
+:doc:`computes <compute>`, :doc:`fixes <fix>`, and :doc:`variables
+<variable>` to be used to define chunk IDs for each atom.  This means
+you can write your own compute or fix to output a per-atom quantity to
+use as chunk ID.  See the :doc:`Modify <Modify>` doc pages for info on
+how to do this.  You can also define a :doc:`per-atom variable
+<variable>` in the input script that uses a formula to generate a chunk
```

Comment:
```suggestion
<variable>` in the input script that uses a formula that generates a chunk
```

Reply:
I think the original formulation is better here.